src/Controller/InspectController.php

```diff
@@ -43,6 +45,40 @@ public function config(ContainerInterface $container, ServerRequestInterface $re
         return $this->responseFactory->createResponse(json_decode($response, null, 512, JSON_THROW_ON_ERROR));
     }
 
+    public function translations(ContainerInterface $container, ServerRequestInterface $request): ResponseInterface
+    {
+        /**
+         * @var $categorySources CategorySource[]
+         */
+        $categorySources = $container->get('tag@translation.categorySource');
+
+        $params = ApplicationState::$params;
+
+        $locales = array_keys($params['locale']['locales']);
+        if ($locales === []) {
+            throw new RuntimeException(
+                'Unable to determine list of available locales. ' .
+                'Make sure that "$params[\'locale\'][\'locales\']" contains all available locales.'
+            );
+        }
+        $messages = [];
+        foreach ($categorySources as $categorySource) {
+            $messages[$categorySource->getName()] = [
+                'messages' => [],
+            ];
+
+            try {
```

Comment:
Why need `try ... catch` here?

Reply:
Some readers may fail while reading. It's unnecessary to have the fault in the debugger, but necessary to have at least something.

Reply:
Suggest for categories/locales with error return null or false value, that mean error. It's allow immediately see problem in debug panel.

Reply:
I'll do it later